idd/Energy+.idd.in

```diff
@@ -50246,6 +50246,161 @@ ZoneHVAC:EquipmentConnections,
+       \type object-list
+       \object-list ScheduleNames
+  A7 ; \field Space Return Air Node 1 Flow Rate Basis Node or NodeList Name
+       \note The optional basis node(s) used to calculate the base return air flow
+       \note rate for the first return air node in this space. The return air flow rate is the sum of the flow rates
+       \note at the basis node(s) multiplied by the Space Return Air Flow Rate Fraction Schedule.
+       \note If this  field is blank, then the base return air flow rate is the total supply
+       \note inlet flow rate to the zone less the total exhaust node flow rate from the space.
+       \type node
+
+SpaceHVAC:ZoneEquipmentSplitter,
+       \extensible:3
+       \memo Distributes the output from a piece of zone equipment to one or more Spaces in the Zone.
+       \memo If any equipment in a zone has a SpaceHVAC:ZoneEquipmentSplitter, then all equipment in the zone must have one.
+       \min-fields 10
+   A1, \field Name
+       \required-field
+       \reference SpaceSplitterNames
+   A2, \field Zone Name
+       \note Must be a controlled zone which has a ZoneHVAC:EquipmentConfiguration object.
+       \required-field
+       \type object-list
+       \object-list ZoneNames
+   A3, \field Zone Equipment Object Type
+       \required-field
+       \type choice
+       \key ZoneHVAC:TerminalUnit:VariableRefrigerantFlow
+       \key ZoneHVAC:AirDistributionUnit
+       \key ZoneHVAC:EnergyRecoveryVentilator
+       \key ZoneHVAC:EvaporativeCoolerUnit
+       \key ZoneHVAC:HybridUnitaryHVAC
+       \key ZoneHVAC:ForcedAir:UserDefined
+       \key ZoneHVAC:FourPipeFanCoil
+       \key ZoneHVAC:OutdoorAirUnit
+       \key ZoneHVAC:PackagedTerminalAirConditioner
+       \key ZoneHVAC:PackagedTerminalHeatPump
+       \key ZoneHVAC:UnitHeater
+       \key ZoneHVAC:UnitVentilator
+       \key ZoneHVAC:VentilatedSlab
+       \key ZoneHVAC:WaterToAirHeatPump
+       \key ZoneHVAC:WindowAirConditioner
+       \key ZoneHVAC:Baseboard:RadiantConvective:Electric
+       \key ZoneHVAC:Baseboard:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:RadiantConvective:Steam
+       \key ZoneHVAC:CoolingPanel:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:Convective:Electric
+       \key ZoneHVAC:Baseboard:Convective:Water
+       \key ZoneHVAC:HighTemperatureRadiant
+       \key ZoneHVAC:LowTemperatureRadiant:VariableFlow
+       \key ZoneHVAC:LowTemperatureRadiant:ConstantFlow
+       \key ZoneHVAC:LowTemperatureRadiant:Electric
+       \key ZoneHVAC:Dehumidifier:DX
+       \key ZoneHVAC:IdealLoadsAirSystem
+       \key ZoneHVAC:RefrigerationChillerSet
+       \key Fan:ZoneExhaust
+       \key WaterHeater:HeatPump:PumpedCondenser
+       \key WaterHeater:HeatPump:WrappedCondenser
+       \key HeatExchanger:AirToAir:FlatPlate
+       \key AirLoopHVAC:UnitarySystem
+   A4, \field Zone Equipment Name
+       \required-field
+       \type object-list
+       \object-list ZoneEquipmentNames
```

Comment:
So what happens if the zone has a PTAC and ZoneExhaust (or any combination of the above list) and this space can only select one of those to use for air flow calculations?

Reply:
> So what happens if the zone has a PTAC and ZoneExhaust (or any combination of the above list) and this space can only select one of those to use for air flow calculations?

You can have more than one SpaceHVAC:ZoneEquipmentSplitter feeding a given space. It's one splitter for every piece of equipment in a zone.

But ZoneExhaust will happen at the zone level (for now) not space, I think, anyway. I'm hoping it works with returns at the zone level as well, so the space return nodes might be unused. Still nailing that down.

Reply:
I might need to remove Fan:ZoneExhaust from the valid types for this - hmmm.